backend/REST-API.js

```diff
@@ -5,20 +5,13 @@ const {
   allPoems
 } = require('./rest-api-test-data.js')
```

Comment:
we need to remove this variable (b/c the test file `rest-api-test-data.js` no longer exists) && instead use the `init_database.js` to import that function instead (see blow)